tests/alive-tv/vector/x86-pshufb.src.ll

```diff
@@ -0,0 +1,287 @@
+; ModuleID = 'x86-pshufb.ll'
+target triple = "x86_64-unknown-unknown"
+
+; These are copied from Transforms/InstCombine/X86/x86-pshufb.ll.
```

Comment:
please don't do this. The copyright belongs to LLVM. We shouldn't bring whole big files in.

Reply:
Oh, I see. I'll write my own tests.